src/test/java/com/epam/tc/hw1/CalculatorDivideTest.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.tc.hw1;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.Test;
+
+public class CalculatorDivideTest {
+    Calculator calculator = new Calculator();
```

Comment:
1. This assignment looks like the initialization of the system under test. Let's use `@Before....` annotated method to do so
2. This method with BeforeSOmething annotation is common for all four test classes, so can be moved to the one parent class (parent for existing four test classes)

Reply:
extended all 4 classes